net-libs/srt/srt-1.5.1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 2018-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-multilib
+
+DESCRIPTION="Secure Reliable Transport (SRT) library and tools"
+HOMEPAGE="https://github.com/Haivision/srt"
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/Haivision/${PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/Haivision/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~ppc ~ppc64 ~riscv -sparc ~x86 ~ppc-macos ~x64-macos"
+fi
+
+LICENSE="MPL-2.0"
+SLOT="0/${PV}"
```

Comment:
I'm not sure this is supposed to be $PV anymore, maybe 1.5 is sufficient now, but would have to check into it. They introduced ABI breakage in a 1.4.x release and then scrambled, but am not sure if it is now settled or not to at least not happen between micro version bumps.

Reply:
So what do you want this PR to do?  Switch to major-minor or keep it as $PV?

Reply:
What's the installed library files like, mainly what does /usr/lib64/libsrt.so symlink to?

Otherwise I guess it's about having some information from upstream materials on this, but unfortunately I don't recall offhand and haven't had a chance to look.

I think it should be OK to change to `SLOT="0/$(ver_cut 1-2)"` (provided libsrt.so symlinks to `libsrt.so.1.5`), because if on next bump we find they broke ABI again, we can just change it to 0/$PV then and have things rebuild as needed anyhow, while if they keep it stable, srt-1.5.2 with SLOT=0/1.5.1 will just be weird.